tsconfig.json

```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "target": "esnext",
-    // "module": "nodenext",
+    // "module": "node next",
```

Comment:
Let's add [`nodenext`](https://www.typescriptlang.org/tsconfig/#node16nodenext-nightly-builds) to known words.